igamt-lite-domain/src/main/java/gov/nist/healthcare/tools/hl7/v2/igamt/lite/domain/ExportConfig.java

```diff
@@ -105,7 +107,8 @@ public static ExportConfig getBasicExportConfig(boolean setAllTrue) {
     defaultConfiguration.setProfileComponentItemsExport(displayAll);
 
     defaultConfiguration.setCodesExport(codeUsageExport);
-
+    defaultConfiguration.setPhinvadsUpdateEmailNotification(false);
+    defaultConfiguration.setEmail(null);
```

Comment:
There should be no email in the config.  Can't we get this from the user account ?